TicTacToe/TicTacToe.cs

```diff
@@ -32,45 +41,76 @@ public static void GetInput()
             int row = int.Parse(Console.ReadLine());
             Console.WriteLine("Enter Column:");
             int column = int.Parse(Console.ReadLine());
+            PlaceMark(row, column);
         }
 
         public static void PlaceMark(int row, int column)
         {
-        // your code goes here
+            if (board[row][column] == " ") {
+                if (playerTurn == "X") {
+                    board[row][column]= "X";
+                }else{
+                    board[row][column] = "0";
+                }
+                count++;
+            }else{
+                Console.WriteLine("That spot has already been chosen. Choose another.");
```

Comment:
If the spot has already been chosen, the way your code is written, the player ends up losing their turn.